auth/integration_test/src/integration_test.cc

```diff
@@ -310,6 +310,7 @@ bool FirebaseAuthTest::WaitForCompletion(
     EXPECT_EQ(result_ptr->additional_user_info.provider_id, provider_id);
     EXPECT_EQ(result_ptr->user.uid(), auth_->current_user().uid());
     EXPECT_TRUE(auth_->current_user().is_valid());
+    EXPECT_TRUE(result_ptr->credential.is_valid());
```

Comment:
Ah, AuthResult.credential is not necessarily valid all the time, even if the request is successful. I think some of the API simply won't have credential information

Reply:
done